src/pages/home/report/ReportActionItem.js

```diff
@@ -258,6 +279,7 @@ class ReportActionItem extends Component {
                             <View
                                 style={StyleUtils.getReportActionItemStyle(
                                     hovered
+                                    || isWhisper
```

Comment:
@grgia Do you think we can use `StyleUtils.getBackgroundAndBorderStyle(themeColors.cardBG)` instead of creating a new style?

Reply:
Oh, you are out, so I am just going to give it a try haha

Reply:
Ahh nvm, I am blind, we are doing that already